users/tests.py

```diff
@@ -28,6 +28,18 @@ def test_guest_can_request_for_otp(self):
         otp_exists = OneTimePassword.objects.filter(mobile=self.user.mobile).exists()
         self.assertTrue(otp_exists)
 
+    def test_otp_not_passed(self):
```

Comment:
so, the name of the test should specify what should happen given a particular set of inputs. So, in this case, a better name could be: `test_verifying_otp_fails_when_otp_not_passed`